packages/components/pie-button/.eslintignore

```diff
@@ -0,0 +1,3 @@
+.turbo
```

Comment:
A note here - I tried adding a single `.eslintignore` to the root of the repo but it didn't seem to get picked up/work